pipelines/.dlt/config.toml

```diff
@@ -57,3 +57,6 @@ access_token_expires_at=1688821881
 
 [sources.workable]
 subdomain="dlthub-test"
+
+[sources.template]
```

Comment:
IMO we should reduce the config.toml usage to a minimum. this is only importatn for people that write pipeliens and test them.